3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,83 @@
+// Age-ify
+const yearOfBirth = 1987;
+let yearFuture = 2045;
+
+// calculating the age
+let age = yearFuture - yearOfBirth;
+console.log(age);
+
+// changing the future year to 2027
+yearFuture = 2027;
+age = yearFuture - yearOfBirth;
+console.log(age);
+
+// log out using string
+console.log(`You will be ${age} in ${yearFuture}`);
+
+//A dog age calculator
+const dogYearOfBirth = 2017;
+let dogYearFuture = 2045;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let dogString = "human";
+
```

Comment:
Good job assigning a default value to dogString. Since we only have two possible values that avoids the use of an else later on.